mgmt_cli_test.py

```diff
@@ -91,7 +91,7 @@ def install_gsutil_dependencies(self, node):
             cmd = dedent("""
                 echo 'deb https://packages.cloud.google.com/apt cloud-sdk main' | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
                 curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
-                sudo apt-get update && sudo apt-get install google-cloud-cli
+                sudo apt-get update && sudo apt-get install -y google-cloud-cli
```

Comment:
please use `node.install_package("google-cloud-cli")` after adding sources & update.
this method contains more options that sometimes might be required and provides retry mechanism in case of e.g. disconnection problem.

Reply:
Fixed.
The testing job has been started: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mikita/job/sct/job/sct-feature-test-backup-gce/4/

Reply:
yes, but it doesn't look like the failure related to this change:
```
AssertionError: The restore task is supposed to fail, since node Node manager-regression-gce-fix-db-node-3a145237-0-2 [34.138.67.112 | 10.142.0.107] lacks the disk space to downloadthe snapshot files
```